website/docs/releases/release-notes.md

```diff
@@ -2,6 +2,21 @@
 title: Release Notes
 ---
 
+## v0.7.0-next.0
+#### cashc compiler
+- :sparkles: Add destructuring assignments, e.g. `bytes2 left, bytes1 right = 0x123456.split(2)`
+- :sparkles: Add multiplication, e.g. `int x = 5 * 5`
+- :sparkles: Add *native* introspection/covenants
```

Comment:
Could explicitly mention adding support for 64 bit ints too?

Reply:
I could, but nothing changed in the compiler to support 64bit ints. I guess I will add it to avoid confusion though.